tests/infer/mcmc/test_mh.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import absolute_import, division, print_function
+
+import pytest
+import torch
+from torch.autograd import Variable
+
+import pyro
+import pyro.infer
+from pyro.distributions import Normal
+from tests.common import TestCase
+from pyro.infer.abstract_infer import Marginal
+from pyro.infer.mcmc.mh import MH, NormalProposal
+from pyro.infer.mcmc.mcmc import MCMC
+from pyro.util import ng_ones, ng_zeros
+
+
+class NormalNormalSamplingTestCase(TestCase):
+
+    def setUp(self):
+
+        pyro.clear_param_store()
+
+        def model():
+            mu = pyro.sample("mu", Normal(Variable(torch.zeros(1)),
+                                          Variable(torch.ones(1))))
+            xd = Normal(mu, Variable(torch.ones(1)), batch_size=50)
```

Comment:
I believe the syntax `batch_size=50` has been replaced with `sample_shape=(50,)`.